### PR TITLE
Added support for --profile flag for docker-compose up and docker-com…

### PR DESCRIPTION
**Pull request by HackTheOxidation**
…pose down.

### CODE REVIEW

File: docker-compose.el [hunk 1]
```diff
@@ -145,6 +145,7 @@
   :man-page "docker-compose down"
   ["Arguments"
    ("o" "Remove orphans" "--remove-orphans")
+   ("p" "Profile" "--profile" read-string)
```
Comment (Silex) [inline docker-compose.el:148]: Needs a space after  `--profile`

File: docker-compose.el [hunk 2]
```diff
@@ -261,6 +262,7 @@
    ("d" "Detach" "-d")
    ("f" "Force recreate" "--force-recreate")
    ("n" "No deps" "--no-deps")
+   ("p" "Profile" "--profile" read-string)
```
Comment (Silex) [inline docker-compose.el:265]: Same here

**Review by Silex — CHANGES_REQUESTED** [review]

**Comment by Silex** [comment]
Hello, please modify your commit with the requested changes, also change the commit message for `Support --profile flag for docker-compose up/down`

File: docker-compose.el (hunk 1 above)
Comment (HackTheOxidation) [reply docker-compose.el]: Good catch. Should be fixed now.

**Comment by Silex** [comment]
Fantastic thanks! :+1:

**Pull request merged**